tcl/main.tcl

```diff
@@ -1232,7 +1233,7 @@ proc CreateGameInfo {} {
           -variable gameInfo(showFEN) -offvalue 0 -onvalue 1 -command updateBoard
 
   .main.gameInfo.menu add checkbutton -label GInfoMarks \
-          -variable gameInfo(showMarks) -offvalue 0 -onvalue 1 -command updateBoard
+          -variable gameInfo(showMarks) -offvalue 0 -onvalue 1 -command { ::board::showMarks .main.board $gameInfo(showMarks); updateBoard }
```

Comment:
Please don't use the ";"
I think that moving the commands each to a separate line would be much more readable